### PR TITLE
Added Slider stories

### PR DESCRIPTION
**Pull request by GomezIvann**

### CODE REVIEW

File: lib/src/slider/Slider.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcSlider label="Xxlarge" margin="xxlarge" />
+    </ExampleContainer>
+    <Title title="Sizes" theme="light" level={2} />
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcSlider label="Large" size="medium" />
+    </ExampleContainer>
```
Comment (raquelarrojo) [inline lib/src/slider/Slider.stories.tsx:162]: Title and slider label here should be `Medium`.

File: lib/src/slider/Slider.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcSlider label="Xlarge" size="large" />
+    </ExampleContainer>
```
Comment (raquelarrojo) [inline lib/src/slider/Slider.stories.tsx:166]: Title and slider label here should be `Large`.

File: lib/src/slider/Slider.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcSlider label="Xxlarge" size="fillParent" />
+    </ExampleContainer>
```
Comment (raquelarrojo) [inline lib/src/slider/Slider.stories.tsx:170]: Title and slider label here should be `FillParent`.

**Comment by raquelarrojo** [comment]
We could also add a test for `labelFormatCallback`.

File: lib/src/slider/Slider.stories.tsx (hunk 1 above)
Comment (GomezIvann) [reply lib/src/slider/Slider.stories.tsx:162]: Done!

File: lib/src/slider/Slider.stories.tsx (hunk 2 above)
Comment (GomezIvann) [reply lib/src/slider/Slider.stories.tsx:166]: Done!

File: lib/src/slider/Slider.stories.tsx (hunk 3 above)
Comment (GomezIvann) [reply lib/src/slider/Slider.stories.tsx:170]: Done!

**Comment by raquelarrojo** [comment]
> Yes, you're right. I think it's a non valuable test.

**Pull request closed without merging**